tests/integration/test_pr_comment.py

```diff
@@ -2600,7 +2600,7 @@ def test_bodhi_update_retrigger_via_dist_git_pr_comment(pagure_pr_comment_added)
 def test_pull_from_upstream_retrigger_via_dist_git_pr_comment(pagure_pr_comment_added):
     pagure_pr_comment_added["pullrequest"]["comments"][0][
         "comment"
-    ] = "/packit pull-from-upstream --resolved-bugs rhbz#123,rhbz#124"
+    ] = "/packit pull-from-upstream --with-pr-config --resolved-bugs rhbz#123,rhbz#124"
```

Comment:
Is there a simple way in this test to assert that we get the bugzillas correctly?

Reply:
it is being tested below via flexmock that checks the arguments to `sync_release`